src/guide/4-initial-conditions/2-initial-properties.rst

```diff
@@ -1,2 +1,47 @@
-Initial Properties
-==================
\ No newline at end of file
+Initialising Agent Variables
+============================
+
+Initialising a Population
+-------------------------
+
+To initialise a population, iterate over the ``AgentVector`` object and set the properties for each agent:
+
+** TODO: not very idiomatic - python should use enumerate I think, and range based for for cpp**
+
+.. tabs::
+
+  .. code-tab:: python
+
+    for i in range(populationSize):
+        instance = population[i];
+        instance.setVariableInt("id", i);
+  
+  .. code-tab:: cpp
+
+    for (unsigned int i = 0; i < populationSize; i++) {
+        AgentVector::Agent instance = population[i];
+        instance.setVariable<int>("id", i);
+    }
+
+Accessing a Single Agent
+------------------------
+
+A single agent can be individually accessed by indexing into the ``AgentVector`` object:
+
+.. tabs::
+
+  .. code-tab:: python
+    
+    # Fetch the 23rd agent
+    instance = population[23]
+
+    # Set its id to 12,000
+    instance.setVariableInt("id", 12000)
```

Comment:
`id` might not be the best example, in the near future id will be managed automatically internally.